pyproject.toml

```diff
@@ -98,6 +98,7 @@ xbundle = "^0.3.1"
 social-auth-core = {extras = ["openidconnect"], version = "^4.4.2"}
 nh3 = "^0.2.14"
 retry2 = "^0.9.5"
+django-user-sessions = "^2.0.0"
```

Comment:
Was this primarily for debugging purposes?  I experimented with removing this and using the usual `MIDDLEWARE` and `SESSION_ENGINE` settings, everything still seemed to work correctly.  I'm a little paranoid about changing those values if not necessary.  If it is primarily for debugging, maybe the `MIDDLEWARE` and `SESSION_ENGINE` values could be updated conditionally depending on whether or not a `KEYCLOAK_DEBUG` setting = True?

Reply:
nvm, it works for 1 browser but logs me out of both if I'm signed into two different ones

Reply:
Actually something still seems off after reverting back to the usual PR:

Flow A): Log into browser 1.  Log into browser 2.  Log out of browser 2.  Correctly logged out of keycloak and mitopen in browser 2 while still logged into both in browser 1.

Flow B): Log into browser 1.  Log into browser 2.  Log out of browser 1.  Incorrectly logged out of keycloak in browser 2 while still logged into mitopen; still logged into browser 1 keycloak url while logged out of mitopen.

Reply:
This is being used to end the Django user's session after receiving a POST request from Keycloak.  Django does not provide a way of ending a user's session without that user making a request as the Django session is normally tied to the request.